Gls estimator (#315)
lumicks/pylake: docs/zrefs.rst

geometric considerations. The greedy algorithm was based on two papers. It initially detects feature points based on
:cite:`sbalzarini2005feature`, followed by line tracing inspired by :cite:`mangeol2016kymographclear`.

Diffusion constant estimation
-----------------------------

> We implemented the unweighted ordinary least squares estimation method (including the optimal number of lag computation) from :cite:`michalet2012optimal`.
> The generalized least squares method was based on :cite:`bullerjahn2020optimal`

Reviewer: ```suggestion
The unweighted ordinary least squares estimation method (including the optimal number of lag computation) is implemented from :cite:`michalet2012optimal`.
The generalized least squares method is based on :cite:`bullerjahn2020optimal`
```
